packages/cli/commands/sandbox/create.js

```diff
@@ -85,7 +53,16 @@ const personalAccessKeyFlow = async (env, account, name) => {
     process.exit(EXIT_CODES.SUCCESS);
   }
 
-  const validName = await promptForAccountNameIfNotSet(updatedConfig, name);
+  let validName = updatedConfig.name;
```

Comment:
@adamawang just giving you the heads up. I tried to DRY things up a little bit by sharing prompts between this command and the auth command. I also added the new `hs auth` logs to the "personalAccessKeyFlow" portion of this command. If you get the chance, I'd love some eyes on the changes in this file to make sure everything makes sense.

Reply:
Everything looks perfect, thanks for the heads up! 🎉